src/scss/components/image-fundation.scss

```diff
@@ -0,0 +1,15 @@
+.image-fundation {
+  &__picture {
+    display: flex;
+    justify-content: center;
+    overflow: hidden;
```

Comment:
You don't need these properties, you just need to define the fixed height for the image container

Reply:
Add the height that it will take in desktop viewport